lib/config/rules/rule-helpers/console-methods.js

```diff
@@ -7,7 +7,6 @@ module.exports = [
   'debug',
   'dir',
   'dirxml',
-  'error',
```

Comment:
Okay, we'll keep `no-console` enabled fully and address the concerns where appropriate